### ISSUE TITLE
If remove the XML complains？

### ISSUE THREAD

**Issue by yimouleng** [danialmoghaddam/SwiftyWebRequestHandler#1]
My server is json instead of XML, if I don't pod SWXMLHash the dependence, the entire library complains?

**Comment by danialmoghaddam** [comment]
yes it complain since the project is built to cover both XML and JSON at the same
